doc/tutorials/parallel.ipynb

```diff
@@ -289,7 +289,7 @@
     "\n",
     "### Benchmark\n",
     "\n",
-    "Finally, let's see if `ProcessPoolExecutor` brings any speedup, compared to sequential processing, to Example 2."
+    "Finally, let's see if `ProcessPoolExecutor` brings any speedup, compared to sequential processing as demonstrated in Example 2."
```

Comment:
well, sequential processing is not demonstrated in Example 2
that's why I added benchmarks that demonstrate the speedup in Example 2

Reply:
Then I didn't get what you wanted to state previously. Let me make another suggestion.

Reply:
I thought you wanted to state that in Example 2 you only used a single worker and want to compare to that... I now removed the reference to Example 2. Hopefully this removes also some potential for confusion.